src/main/java/net/dv8tion/jda/core/entities/impl/SystemMessage.java

```diff
@@ -31,7 +31,7 @@ public SystemMessage(
         String content, String nonce, User author, OffsetDateTime editTime,
         List<MessageReaction> reactions, List<Attachment> attachments, List<MessageEmbed> embeds)
     {
-        super(id, channel, type, fromWebhook, mentionsEveryone, tts, pinned, content, nonce, author, editTime, reactions, attachments, embeds);
+        super(id, channel, type, fromWebhook, mentionsEveryone, tts, pinned, content, nonce, author, editTime, reactions, attachments, embeds, null);
```

Comment:
Doesn't this seem inconsistent? All other parameters are also present as parameters in SystemMessage's constructor, why move away from that consistency, we parse it either way

Reply:
I did some research and SystemMessages do not have activity so i did not add it there.

Reply:
They also do not have embeds, however we parse it and should support it if possible

Reply:
It does now i do test and hunt more ActivityTypes atm so the next commit may appear later.

Reply:
There must be a type for RPC invites such as spectating and join game